apps/controllo-ore-x/src/app/_core/interfaces/i-menu-sections.interface.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Interface for menu sections
+ */
+export interface IMenuSections {
```

Comment:
```suggestion
export interface IMenuSection {
```
Ho notato ora che era al plurale